site/content/posts/2021-06-21-windows-e2e.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Running E2E Tests On Windows Clusters
+image: /img/sonobuoy.svg
+excerpt: Sonobuoy has new plugins which make running the Kubernetes E2E tests on Windows clusters much more simple.
```

Comment:
Could be "Sonobuoy makes running Kubernetes E2E tests on Windows clusters simpler with plugins"